SECP-service/src/main/resources/assets/app/index.html

```diff
@@ -1,36 +1,43 @@
 <!doctype html>
 <html lang="en" ng-app="SECP">
+
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>SECP</title>
   <link href="lib/bootstrap-css/css/bootstrap.css" rel="stylesheet">
-  <link href="lib/jquery-ui/themes/smoothness/jquery-ui.css" rel="stylesheet"/>
+  <link href="lib/jquery-ui/themes/smoothness/jquery-ui.css" rel="stylesheet" />
   <link href="css/app.css" rel="stylesheet">
 </head>
+
 <body>
- <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
-   <div class="container">
-     <div class="navbar-header">
-       <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse">
-         <span class="sr-only">Toggle navigation</span>
-         <span class="icon-bar"></span>
-         <span class="icon-bar"></span>
-         <span class="icon-bar"></span>
-       </button>
-       <a class="navbar-brand" href="#">SECP</a>
-     </div>
-
-     <!-- Collect the nav links, forms, and other content for toggling -->
-     <div class="collapse navbar-collapse navbar-ex1-collapse">
-       <ul class="nav navbar-nav">
-         
-       </ul>
-     </div>
-     <!-- /.navbar-collapse -->
-   </div>
-   <!-- /.container -->
+  <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
+    <div class="container">
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse">
+          <span class="sr-only">Toggle navigation</span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </button>
+        <a class="navbar-brand" href="#/">SECP</a>
+      </div>
+
+      <!-- Collect the nav links, forms, and other content for toggling -->
+      <div class="collapse navbar-collapse navbar-ex1-collapse">
+        <ul class="nav navbar-nav">
+          <li>
+            <a href="#/login">Login</a>
+          </li>
+          <li>
+            <a href="#/register">Register</a>
```

Comment:
I do not think the register and login belongs on the navbar in the index.html file. If the user is logged in, the "login and register" link would still be on the navbar. Please put it in home.html.

Reply:
I think login should be there in the navbar, but also a link to it in the homepage would be good as well. Thats for the homepage view implementation, which isn't the focus of this. I agree that register shouldn't be in the navbar, but it also shouldn't be in homepage. Registering users should only be done by admin.